docs/tutorials/motion/plan-motion-with-arm-gripper.md

```diff
@@ -0,0 +1,561 @@
+
+Because the new gripper component is "attached" (with the parent specification in the Frame) to `myArm`, we can produce motion plans using `myGripper` instead of `myArm` 
+
+The last library you must import is the `gripper` library.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+Do this by adding `from viam.components.gripper import Gripper` to your import list.
+
+```python {class="line-numbers linkable-line-numbers"}
+my_gripper_resource = Gripper.get_resource_name("myGripper")
+my_gripper_component = Gripper.from_robot(robot, "myGripper")
+
+# This will move the gripper in the -Z direction with respect to its own reference frame
+gripper_pose_rev = Pose(x=0.0, y=0.0, z=-100.0, o_x=0.0, o_y=0.0, o_z=1.0, theta=0.0)
+gripper_pose_rev_in_frame = PoseInFrame(reference_frame=my_gripper_resource.name, pose=gripper_pose_rev) # Note the change in frame name
+
+await motion_srv.move(component_name=gripper_resource, destination=gripper_pose_rev_in_frame, world_state=world_state)
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+Do this by adding `"go.viam.com/rdk/components/gripper"` to your import list.
+
+```go {class="line-numbers linkable-line-numbers"}
+gripperName := "myGripper"
+gripperResource := gripper.Named(gripperName)
+
+// This will move the gripper in the -Z direction with respect to its own reference frame
+gripperPoseRev := spatialmath.NewPose(
+  r3.Vector{X: 0.0, Y: 0.0, Z: -100.0},
+  &spatialmath.OrientationVectorDegrees{OX: 0.0, OY: 0.0, OZ: 1.0, Theta: 0.0},
+)
+gripperPoseRevInFrame := referenceframe.NewPoseInFrame(gripperName, gripperPoseRev) // Note the change in frame name
+
+_, err = motionService.Move(context.Background(), gripperResource, gripperPoseRevInFrame, worldState, nil, slam.Named(""), nil)
+if err != nil {
+  logger.Fatal(err)
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+For the gripper pose, you can change the reference frame information to consider other objects or user-generated frames that exist in the Frame System.
+Specifying other reference frames is an easy way to move with respect to those frames.
+For example, you can specify a pose that is 100 millimeters above the table obstacle featured earlier in this tutorial.
+You do not need to calculate that exact pose with respect to the **arm** or **world**.
+You must only provide the object name (instead of the `gripperName` you saw in the code samples above) when making the `NewPoseInFrame` to pass into the `Move` function.
+This has implications for how motion is calculated, what magnitude and direction is considered, and what the mot to consider when planning.
+
+## Next Steps and References
+
+<!-- TODO: Content below struck out for the moment, saved to point at the next tutorial "Plan Motion with an Arm and with a Gripper" -->
+<!--
+If you would like to continue onto specific complex motion constraints, go to the next tutorial in this series: "[Put Constraints on a Motion Plan](/tutorials/motion/plan-motion-with-arm-gripper/)".
+-->
+
```

Comment:
You've reached the end of this tutorial but if you want to learn more, have a look at our [other tutorial](../../). 

If you encounter any issues while using the Motion Service or otherwise want to chat about the robot you're building, join us in the [Community Discord](https://discord.gg/viam).

Reply:
Added the top bit, should I take out the `social.md` snippet and replace with the bottom bit?